cmd/shell.go

```diff
@@ -0,0 +1,46 @@
+// Copyright © 2016 Prometheus Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+func runCommand(name string, arg ...string) error {
+	if viper.GetBool("verbose") {
+		cmdText := name + " " + strings.Join(arg, " ")
+		fmt.Fprintln(os.Stderr, " > ", cmdText)
```

Comment:
Nevermind, this is how `set -x` is working. I would prefer a `+ ` prefix instead of ` > `, like a regular command trace.